src/meta/meta_options.cpp

```diff
@@ -161,6 +161,11 @@ void meta_options::initialize()
     _lb_opts.only_move_primary = dsn_config_get_value_bool(
         "meta_server", "only_move_primary", false, "only try to make the primary balanced by move");
 
+    partition_guardian_type = dsn_config_get_value_string("meta_server",
+                                                          "partition_guardian_type",
+                                                          "partition_guardian",
+                                                          "partition guardian provider");
+
```

Comment:
One of the `partition_guardian_type` is "partition_guardian", does it have another type?

Reply:
There is another type name dummy_partition_guardian in unit test.